components/dashboard/src/components/forms/CheckboxInputField.tsx

```diff
@@ -70,7 +70,7 @@ export const CheckboxInput: FC<CheckboxInputProps> = ({
             <div className="flex flex-col">
                 <span
                     className={classNames(
-                        "text-sm font-semibold cursor-pointer",
+                        "text-md font-semibold cursor-pointer",
```

Comment:
**question**: Curious what led to this change?

| BEFORE | AFTER |
|-|-|
| <img width="550" alt="check-before" src="https://user-images.githubusercontent.com/120486/228930156-4d90fc7c-49d1-443f-8b9d-60cf1e4ee4f2.png"> | <img width="550" alt="check-after" src="https://user-images.githubusercontent.com/120486/228930164-d7d99789-29a0-4981-b75c-541a241250ec.png"> |

Reply:
@gtsiolis This change is made to maintain style consistency across all the `<CheckboxInput />` components used on different dashboard pages. The small font looks good for UX on the Onboarding flow page but doesn't look quite good on other pages that initially used the old `<CheckBox />` component (which uses the medium-sized font).

Reply:
Ah, good point, @Devansu-Yadav! Yeah, the smaller font size on the onboarding was probably changed because of the many options there which ideally would be in a disclosure or a combobox dropdown. Sounds good to leave this change as is. Cc @selfcontained

Reply:
I'd prefer we leave it as `text-sm` for now so we don't effect the onboarding UI with this change. Other occurrences of checkboxes shifting to a smaller font feel preferred imo to bumping up the vertical height on that already quite long onboarding UI w/ all the questions. We can iterate and explore bumping up the size in it's own PR where we also address the onboarding UI to keep text smaller there if we want.